website/content/partials/components/builder-pack.mdx

```diff
@@ -67,6 +67,56 @@ The entrypoint binary is what provides extended functionality such as logs and e
 - Type: **bool**
 - **Optional**
 
+#### ignore
```

Comment:
This is kind of a mess, are we OK with that?

Reply:
I couldn't figure it out, so I copy pasted the text into the mdx file 😖 I'll open an issue to actually fix it, but for now I'd say good enough for getting the website looking nice.